### PR TITLE
Refactored and deployed event schema stack

### PR DESCRIPTION
**Pull request by victorskl**
* Finalised SequenceRunStateChange event schema - trim & case
* Relocated event schemas directory into docs
* Deployed schema stack

Related #257

### CODE REVIEW

**Comment by victorskl** [comment]
I will need to get merge this refactor PR first, before following up next for workflow event schema. At the same time, I need your input and review. So, will be waiting.

**Review by williamputraintan — APPROVED** [review]
Hmmm, I just have the preference of having as close as possible to the code so perhaps stay as it is in the config folder 😅 (or maybe at the stack folder) ? I think the `docs/` should be just a bunch of explanation on how to do what instead of relying some actual content. 

Or it might be just me not used to it, so I'm happy either way 😁

File: config/constants.ts [hunk 1]
```diff
@@ -76,7 +76,7 @@ export const rdsMasterSecretName = 'orcabus/master-rds'; // pragma: allowlist se
 validateSecretName(rdsMasterSecretName);
 
 // Other constants that exist in the SharedStack
-export const regName = 'OrcaBusSchemaRegistry';
+export const regName = 'orcabus.events';
```
Comment (reisingerf) [inline config/constants.ts:79]: I don't think we have other registries planned, but can we be a bit more specific?
E.g. `eventRegName` or `schemaRegName`

File: docs/event-schemas/srm/SequenceRunStateChange.json [hunk 2]
```diff
@@ -56,48 +54,44 @@
...
         "required": [
           "id",
-          "instrument_run_id"
+          "instrumentRunId",
+          "runVolumeName",
+          "runFolderPath",
+          "runDataUri",
+          "sampleSheetName",
+          "startTime",
+          "endTime",
```
Comment (reisingerf) [inline docs/event-schemas/srm/SequenceRunStateChange.json:63]: You are keeping one record for each SequenceRun, right?
(i.e. there are defined start/end times and no intermediate state-bound times)

**Review by reisingerf — APPROVED** [review]
Minor comments. 
Otherwise LGTM

File: config/constants.ts (hunk 1 above)
Comment (victorskl) [reply config/constants.ts:79]: Ok, I will change it to `schemaRegistryName`

File: docs/event-schemas/srm/SequenceRunStateChange.json (hunk 2 above)
Comment (victorskl) [reply docs/event-schemas/srm/SequenceRunStateChange.json:63]: Yes, that right. One record for each run; unique by `instrumentRunId`.

File: config/constants.ts (hunk 1 above)
Comment (victorskl) [reply config/constants.ts]: Fixed with https://github.com/umccr/orcabus/commit/efd9a5b5fac29a669c8d34e7e4b49dd692585f49

**Comment by victorskl** [comment]
> preference of having as close as possible to the code

Good point, Will. Schema is a bit tricky. Schema is a contract documentation by definition. Putting under docs section emphasizes this point. We might probably using some doc generator on these schemas to present them to user or observer, at some point down track. So, these folks won't need to dig through our code base, too much.

**Comment by victorskl** [comment]
Alexis; Ray; going to merge this now. Comment if any.

**Pull request merged**